chat-client/src/client/chat.ts

```diff
@@ -1,4 +1,19 @@
 /* eslint-disable prefer-const */
+import {
+    AUTH_FOLLOW_UP_CLICKED,
+    AuthFollowUpClickedParams,
+    ERROR_MESSAGE,
+    ErrorMessage,
+    GENERIC_COMMAND,
+    GenericCommandMessage,
+    INSERT_TO_CURSOR_POSITION,
+    InsertToCursorPositionParams,
+    SEND_TO_PROMPT,
+    SendToPromptMessage,
+    TAB_ID_RECEIVED,
```

Comment:
I should have raised it in previous PR: is this notification really needed on UI? Depending on the server logic, it can be needed on server or not needed at all imo.

Reply:
Yeah, we can also leave this to separate PR. Logic around that event does not look straightforward.